catlearn/tensor_utils.py

```diff
@@ -147,6 +147,12 @@ def ones_like(
     return full_like(1.0, tensor, shape)
 
 
+def one_hot(label: int, nb_labels):
```

Comment:
Redundant with catlearn/utils/one_hot()

Reply:
Actually this one came as a replacement of an other one which was alone in a file (don't remember which). Wanted to clean a bit^^ didn't pay attention there was an other one you had added.
So yeah we'll keep only one (I guess the one in tensor_utills as I think it indeed makes more sense).